.github/workflows/build-test.yml

```diff
@@ -10,7 +10,28 @@ on:
     branches:
       - '**'
 jobs:
+  pre_build_test:
+    runs-on: ubuntu-latest
+    # Map a step output to a job output
+    outputs:
+      should_skip: ${{ steps.skip_check.outputs.should_skip }}
+    steps:
+      - id: skip_check
+        uses: fkirc/skip-duplicate-actions@master
+        with:
+          # Never allow two runs of this workflow to exist
+          concurrent_skipping: 'always'
```

Comment:
Does this skip the prior run or the latter?

Reply:
This is not clear to me from the docs, so I'll have to experiment